internal/schemas.ts

```diff
@@ -790,6 +790,36 @@ const CompressedImage: FoxgloveMessageSchema = {
   ],
 };
 
+const CompressedVideo: FoxgloveMessageSchema = {
+  type: "message",
+  name: "CompressedVideo",
+  description: "A single frame of a compressed video bitstream",
+  fields: [
+    {
+      name: "timestamp",
+      type: { type: "primitive", name: "time" },
+      description: "Timestamp of image",
```

Comment:
Why do we keep the "image" terminology here? Should this be "video frame" instead or something like that?